test/features/user_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule Syndeo.Feature.UserTest do
+  use Syndeo.FeatureCase
+
+  test "create a user and it shows" do
+    user = build(:user) |> save
+    sign_in_and_visit "/admin/users", user
+    click_on "New user"
+
+    fill_in "user", "email", with: "me@example.com"
+    fill_in "user", "password", with: "password"
+    submit()
+
+    navigate_to "/admin/users"
+    assert text_for_role("user") =~ "me@example.com"
+  end
+
+  test "update existing user" do
+    user = build(:user) |> save
```

Comment:
I know this is minor but I would specify the email here. If you don't, there is no guarantee that the seeds don't have `jason@example.com` when you started this test and so you didn't actually change anything.